libs/model/src/policies/handlers/notifyReferrerSignedUp.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  EventContext,
+  EventHandler,
+  notificationsProvider,
+  WorkflowKeys,
+} from '@hicommonwealth/core';
+import z from 'zod';
+import { models } from '../../database';
+
+export const notifyReferrerSignedUp: EventHandler<
+  'SignUpFlowCompleted',
+  z.ZodBoolean
+> = async (event: EventContext<'SignUpFlowCompleted'>) => {
+  const { user_id, referred_by_address } = event.payload;
+  if (!referred_by_address) return;
+
+  const referrer = await models.Address.findOne({
+    where: { address: referred_by_address },
+    attributes: ['user_id'],
+  });
+  if (!referrer?.user_id) return;
+
+  const provider = notificationsProvider();
+  const res = await provider.triggerWorkflow({
+    key: WorkflowKeys.ReferrerSignedUp,
+    users: [{ id: referrer.user_id.toString() }],
+    data: { referee_user_id: user_id },
```

Comment:
Do we not need more data about the referee e.g. avatar, username, etc?

Reply:
Adding profile details, but we'll probably iterate over the required attributes after looking at the final workflows